### DISCUSSION TITLE
プロフィールのフォロー中タグ一覧に質問タグのフォローも表示・解除できるようにして欲しい

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#266] [要望・アイデア・不具合]
プロフィールページの「フォロー中のタグ」をクリックするとフォローしているタグ一覧が表示されますが、ここには記事タグのフォローだけが表示されているようで、フォロー解除しても記事のフォローしか解除できませんでした。
質問タグのフォローも表示するようにして、解除もできるようにしてはいかがでしょうか。

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご意見ありがとうございます🙏
マイページをもっと役立つ情報設計にアップデートしていきたいと思っておりますので、
その際に今回頂いた内容も検討させていただきます🙇
